Modules/HypnokinkModule/Scenes/HK_RushRepeatConversation.gd

```diff
@@ -0,0 +1,388 @@
+
+		addButton("Deal", "This will hurt", "deal")
+		addButton("No deal", "You don't like your chances", "no_deal")
+	if(state == "deal"):
+		playAnimation(StageScene.Duo, "stand", {npc="Rush"})
+		saynn("His attitude sobers. He flexes, and enters a classic boxing stance.")
+
+		saynn("[say=Rush]Alrigh', but don' go sayin' I didn't warn ya. Put 'em up.[/say]")
+
+		saynn("You try to get ready for what's certainly going to hurt in the morning. If you're lucky.")
+
+		addButton("Put 'em up", "You weren't using all those ribs anyway", "put_em_up")
+	if(state == "no_deal"):
+		playAnimation(StageScene.Duo, "stand", {npc="Rush"})
+		saynn("He claps you on the back, sending you stumbling into the wall.")
+
+		saynn("[say=Rush]Yeh tha's right. Good on ya, broken bones ain't no fun for no one.[/say]")
+		
+		addButton("Back", "", "")
+	if(state == "if_won"):
+		playAnimation(StageScene.Duo, "stand", {npc="Rush", npcAction="defeat"})
+		saynn("Rush stumbles and falls to one knee.")
+		
+		saynn("[say=Rush]Whoo. Ya got a mean kick to ye, pipsqueak.[/say]")
+
+		saynn("Then he rises back to his feet, bruised but unwavering.")
+		
+		saynn("[say=Rush]Ya got me good. Deal's a deal. Tell four-eyes he's good. Until I need more credits.[/say]")
+		
+		addButton("Done here", "Good enough for now", "leave_victorious")
+		if(getFlag("HypnokinkModule.RushDealStruckAtLeastOnce")):
+			saynn("A thought occurs to you. If you were to knock Rush out, there might be a more permanent solution to this problem.")
+			if(GM.pc.getInventory().hasItemID("inmatewristcuffs") and GM.pc.getInventory().hasItemID("inmateanklecuffs")):
+				addButton("Solution", "Keep fighting", "put_em_up_again")
+			else:
+				addDisabledButton("Solution", "Keep fighting\n\nYou'll need a set of wrist and ankle cuffs for this")
+		
+	if(state == "if_lost"):
+		playAnimation(StageScene.Duo, "defeat", {npc="Rush"})
+		saynn("With one last mighty kick, Rush sends you sprawling on the floor.")
+		
+		saynn("[say=Rush]Hah! Tha's right![/say]")
+		
+		saynn("Rush advances onto your prone form. You curl up, anticipating a very unpleasant next few minutes...")
+		
+		saynn("Instead, he pulls you back onto your feet.")
+		
+		saynn("[say=Rush]Not bad, pipsqueak! Ya wanna throw down like tha' again sometime?[/say]")
+		
+		saynn("He brushes the dust off your shoulders, making you wince as his rough hands glide over your bruises.")
+		
+		saynn("[say=Rush]Ya know where t' find me. Deal's still on th' table![/say]")
+		
+		saynn("[say=pc]I don't- Yeah, maybe. Sometime else.[/say]")
+		
+		saynn("You decide to cut your losses and slink off to lick your wounds elsewhere.")
+		
+		addButton("Ow", "", "endthescene")
+		
+	if(state == "leave_victorious"):
+		saynn("You decide to quit while you're ahead and leave to report the good news.")
+		
+		addButton("Leave", "", "endthescene")
+		
+	if(state == "if_won_serious"):
+		
+		addButton("Heave", "Better get moving", "drag_rush")
```

Comment:
There is no text here if you win twice, only the button. Should it be Heave or Leave?

Reply:
Fix'd lack of text.